presto-product-tests/src/main/java/com/facebook/presto/tests/hive/TestSqlStandardAccessControlChecks.java

```diff
@@ -53,6 +56,11 @@ public void testAccessControlSelect()
 
         aliceExecutor.executeQuery(format("GRANT SELECT ON %s TO bob", tableName));
         assertThat(bobExecutor.executeQuery(format("SELECT * FROM %s", tableName))).hasNoRows();
+
+        assertThat(() -> bobExecutor.executeQuery(format("SELECT * FROM %s", viewName)))
+                .failsWithMessage(format("Access Denied: Cannot select from table default.%s", viewName));
```

Comment:
@rschlussel2 @electrum i guess you're aware we are saying "Cannot select from table" for views?

Reply:
I don't think this message is confusing to a user though.  Because they just look at the query and see what "tables" they try to access.

Reply:
@rschlussel2 thanks. If you're OK with it, please accept the PR